example/cumulus-tf/main.tf

```diff
@@ -216,6 +215,15 @@ module "thin_egress_app" {
   vpc_subnet_ids             = var.lambda_subnet_ids
 }
 
+resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscription_filter" {
```

Comment:
This isn't required at all, but just a thought I had, what do you think about moving all the TEA related resources to a separate `tea.tf` file? It might be better organized than cluttering up this file.

This would include all the resources defined from line 173 to 226 in this file

Reply:
For posterity, we discussed on slack that it is probably better to keep resources and module in `main.tf`